docs/src/pages/Conformance.md

```diff
@@ -124,30 +123,67 @@ The whole system history interaction can be used to obtain a list of changes (cr
 
-To return all changes that have occurred since a known point in time, use the `_since` query parameter:
+There are three possible traversal paths through the data which are requested using the `_sort` request parameter:
+
+1. `_sort=-_lastUpdated` [Default] Decreasing time - most recent changes first
+2. `_sort=_lastUpdated` Increasing time - oldest changes first
+3. `_sort=none` Increasing id order - oldest changes first
+
+Option 1 is useful for users interested in finding the most recent changes that have been applied to the server. Option 2 is useful for system-to-system synchronization because it represents the change history of the server. Option 3 is also useful for system-to-system synchronization cases, but guarantees uniqueness across pages when following the `next` links (versus option 2 which may repeat entries at the time window boundary when multiple resources share the same modification timestamp).
+
+To return all changes that have occurred since a known point in time, use both `_since` and `_sort=_lastUpdated` query parameters:
+
+```
+    curl -k -u '<username>:<password>' 'https://<host>:<port>/fhir-server/api/v4/_history?_since=2021-02-21T00:00:00Z&_sort=_lastUpdated'
+```
+
+To return all changes that have occurred before a known point in time, use `_before` and `_sort=-_lastUpdated` query parameters:
 
 ```
-    curl -k -u '<username>:<password>' 'https://<host>:<port>/fhir-server/api/v4/_history?_since=2021-02-21T00:00:00Z'
+    curl -k -u '<username>:<password>' 'https://<host>:<port>/fhir-server/api/v4/_history?_before=2021-02-21T00:00:00Z&_sort=-_lastUpdated'
 ```
 
-By default, the returned bundle will contain up to 100 resource references. The number of resources can be increased (up to 1000) using the `_count` parameter. Specifying `_count` values larger than 1000 will return no more than 1000 resource references:
+To return changes within a known window of time, specify both the `_since` and `_before` query parameters and choose either ascending or descending sort. 
+
+When sorting with ascending time, the `_since` parameter in the `next` link is calculated to fetch the next page of resources and the `_before` parameter (if given) remains constant. When sorting with descending time, the `_before` parameter in the `next` link is calculated to fetch the next page of resources and the `_since` parameter (if given) remains constant.
+
+When sorting using ascending or descending time, the `next` link also includes a value for `_lastChangeId`. This value is used to exclude this resource change record from the next result page. This is required because multiple resources may share the same `_lastUpdated` time, so filters for `_since` and `_before` must always be inclusive which can lead to duplicates across result page boundaries.
```

Comment:
`_lastChangeId` felt a bit ambiguous to me because I felt it could be interpretted as "the last change id that the client would like to receive in their request" whereas it actually is "the last change id the client received in the previous request"

Robin and I brainstormed  alternative names:

- _lastChangeIdReveived
- _pagingRef
- _pagingMarker
- _pageMarker
- _pageMark
- _changeIdMarker
- _changeIdMark
- _idMark
- _mark

We finally settled on `_changeIdMarker`

Reply:
done